test/functorch/test_vmap.py

```diff
@@ -3730,14 +3730,17 @@ def test_vmap_exhaustive(self, device, dtype, op):
         xfail('linalg.lu', ''),
         skip('linalg.ldl_solve', ''),
         skip('_softmax_backward_data'),
+        # AssertionError: Tensor-likes are not equal!
+        # Issue: https://github.com/pytorch/pytorch/issues/70904
+        xfail('bitwise_left_shift', device_type='cpu'),
+        decorate('bitwise_right_shift', device_type='cpu',
+                 decorator=expectedFailureIf(not (IS_MACOS and IS_X86))),
```

Comment:
Thanks for reviewing, @Chillee!

As per [CI failure](https://hud.pytorch.org/pytorch/pytorch/commit/e54f7b3edde356c97c99706942f4b32a5a5ba475), I see the `bitwise_right_shift` test passes with the `macOS` platform in `x86_64` architecture. while testing cross-platform. Earlier I xfailed with the CPU device type.

Please correct me if I'm wrong. Thanks for the question! :)